TwilioVerifyDemo/TwilioVerifyDemo/Common/KeyChainAccessGroupHelper.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  KeyChainAccessGroupHelper.swift
+//  TwilioVerifyDemo
+//
+//  Created by Alejandro Orozco Builes on 7/01/22.
+//  Copyright © 2022 Twilio. All rights reserved.
+//
+
+import Foundation
+import Security
+
+public struct KeyChainAccessGroupInfo {
+  public var prefix: String
+  public var keyChainGroup: String
+  public var rawValue: String
+}
+
+open class KeyChainAccessGroupHelper {
+
+  public class func getAccessGroupInfo() -> KeyChainAccessGroupInfo? {
+    let query: [String:Any] = [
+      kSecClass as String : kSecClassGenericPassword,
+      kSecAttrAccount as String : "detectAppIdentifierForKeyChainGroupIdUsage",
+      kSecAttrAccessible as String: kSecAttrAccessibleAlwaysThisDeviceOnly,
+      kSecReturnAttributes as String : kCFBooleanTrue
+    ]
+
+    var dataTypeRef: AnyObject?
+    var status = withUnsafeMutablePointer(to: &dataTypeRef) { SecItemCopyMatching(query as CFDictionary, UnsafeMutablePointer($0)) }
+
+    if status == errSecItemNotFound {
+      let createStatus = SecItemAdd(query as CFDictionary, nil)
+      guard createStatus == errSecSuccess else { return nil }
+      status = withUnsafeMutablePointer(to: &dataTypeRef) { SecItemCopyMatching(query as CFDictionary, UnsafeMutablePointer($0)) }
+    }
+
+    guard status == errSecSuccess else { return nil }
+
+    let accessGroup = ((dataTypeRef as! [AnyHashable: Any])[(kSecAttrAccessGroup as String)] as! String)
+
+    if accessGroup.components(separatedBy: ".").count > 0 {
+      let components = accessGroup.components(separatedBy: ".")
+      let prefix = components.first!
+      let elements = components.filter() { $0 != prefix }
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/3d084cdf558fe2c3c223d0f0881d860cd43299b1/TwilioVerifyDemo/TwilioVerifyDemo/Common/KeyChainAccessGroupHelper.swift#L44"></span>When using trailing closures, empty parentheses should be avoided after the method call.<br />
<code>empty_parentheses_with_trailing_closure</code> <code>KeyChainAccessGroupHelper.swift:44</code></td>
    </tr>
  </tbody>
</table>